File: csharp/src/IO.Swagger/Model/VaultCheck.cs
using System;
using System.Linq;
using System.IO;
using System.Text;
using System.Text.RegularExpressions;
using System.Collections;
using System.Collections.Generic;
using System.Collections.ObjectModel;
using System.Runtime.Serialization;
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;
using System.ComponentModel.DataAnnotations;
using SwaggerDateConverter = IO.Swagger.Client.SwaggerDateConverter;

namespace IO.Swagger.Model
{
    [DataContract]
    public partial class VaultCheck :  IEquatable<VaultCheck>, IValidatableObject
    {
        public VaultCheck(Response response = default(Response), int? checkID = default(int?), bool? main = default(bool?), string checkRouting = default(string), string checkAccount = default(string), string checkNumber = default(string), string checkFirstName = default(string), string checkLastName = default(string), string note = default(string), string maskedRouting = default(string), string maskedAccount = default(string), string accountType = default(string))
        {
            this.Response = response;
            this.CheckID = checkID;
            this.Main = main;
            this.CheckRouting = checkRouting;
            this.CheckAccount = checkAccount;
            this.CheckNumber = checkNumber;
            this.CheckFirstName = checkFirstName;
            this.CheckLastName = checkLastName;
            this.Note = note;
            this.MaskedRouting = maskedRouting;
            this.MaskedAccount = maskedAccount;
            this.AccountType = accountType;
        }

        [DataMember(Name="Response", EmitDefaultValue=false)]
        public Response Response { get; set; }

        [DataMember(Name="CheckID", EmitDefaultValue=false)]
        public int? CheckID { get; set; }

        [DataMember(Name="Main", EmitDefaultValue=false)]
        public bool? Main { get; set; }

        [DataMember(Name="CheckRouting", EmitDefaultValue=false)]
        public string CheckRouting { get; set; }

        [DataMember(Name="CheckAccount", EmitDefaultValue=false)]
        public string CheckAccount { get; set; }

        [DataMember(Name="CheckNumber", EmitDefaultValue=false)]
        public string CheckNumber { get; set; }

        [DataMember(Name="CheckFirstName", EmitDefaultValue=false)]
        public string CheckFirstName { get; set; }

        [DataMember(Name="CheckLastName", EmitDefaultValue=false)]
        public string CheckLastName { get; set; }

        [DataMember(Name="Note", EmitDefaultValue=false)]
        public string Note { get; set; }

        [DataMember(Name="MaskedRouting", EmitDefaultValue=false)]
        public string MaskedRouting { get; set; }

        [DataMember(Name="MaskedAccount", EmitDefaultValue=false)]
        public string MaskedAccount { get; set; }

        [DataMember(Name="AccountType", EmitDefaultValue=false)]
        public string AccountType { get; set; }
    }
}

File: CodeSamples/nodejs/src/model/VaultCheck.js
/* eslint-disable no-prototype-builtins */
import ApiClient from '../ApiClient';
import Response from './Response';

export default class VaultCheck {
    constructor() {
        /**
    * @member {module:model/Response} Response
    */
    this.Response = new Response();
    /**
    * @member {Number} CheckID
    */
    this.CheckID = 0;
    /**
    * @member {Boolean} Main
    */
    this.Main = false;
    /**
    * @member {String} CheckRouting
    */
    this.CheckRouting = '';
    /**
    * @member {String} CheckAccount
    */
    this.CheckAccount = '';
    /**
    * @member {String} CheckNumber
    */
    this.CheckNumber = '';
    /**
    * @member {String} CheckFirstName
    */
    this.CheckFirstName = '';
    /**
    * @member {String} CheckLastName
    */
    this.CheckLastName = '';
    /**
    * @member {String} Note
    */
    this.Note = '';
    /**
    * @member {String} MaskedRouting
    */
    this.MaskedRouting = '';
    /**
    * @member {String} MaskedAccount
    */
    this.MaskedAccount = '';
    /**
    * @member {String} AccountType
    */
    this.AccountType = '';
    }

    static constructFromObject(data, obj) {
        if (data) {
            obj = obj || new VaultCheck();

            if (data.hasOwnProperty('Response')) {
                obj['Response'] = Response.constructFromObject(data['Response']);
            }
            if (data.hasOwnProperty('CheckID')) {
                obj['CheckID'] = ApiClient.convertToType(data['CheckID'], 'Number');
            }
            if (data.hasOwnProperty('Main')) {
                obj['Main'] = ApiClient.convertToType(data['Main'], 'Boolean');
            }
            if (data.hasOwnProperty('CheckRouting')) {
                obj['CheckRouting'] = ApiClient.convertToType(data['CheckRouting'], 'String');
            }
            if (data.hasOwnProperty('CheckAccount')) {
                obj['CheckAccount'] = ApiClient.convertToType(data['CheckAccount'], 'String');
            }
            if (data.hasOwnProperty('CheckNumber')) {
                obj['CheckNumber'] = ApiClient.convertToType(data['CheckNumber'], 'String');
            }
            if (data.hasOwnProperty('CheckFirstName')) {
                obj['CheckFirstName'] = ApiClient.convertToType(data['CheckFirstName'], 'String');
            }
            if (data.hasOwnProperty('CheckLastName')) {
                obj['CheckLastName'] = ApiClient.convertToType(data['CheckLastName'], 'String');
            }
            if (data.hasOwnProperty('Note')) {
                obj['Note'] = ApiClient.convertToType(data['Note'], 'String');
            }
            if (data.hasOwnProperty('MaskedRouting')) {
                obj['MaskedRouting'] = ApiClient.convertToType(data['MaskedRouting'], 'String');
            }
            if (data.hasOwnProperty('MaskedAccount')) {
                obj['MaskedAccount'] = ApiClient.convertToType(data['MaskedAccount'], 'String');
            }
            if (data.hasOwnProperty('AccountType')) {
                obj['AccountType'] = ApiClient.convertToType(data['AccountType'], 'String');
            }
        }
        return obj;
    }
}
